Assets/Scripts/AirRune.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AirRune : MonoBehaviour
+{
+    public float force = 20;
+
+    public ArrayList inside;
+
+    void Start()
+    {
+    	inside = new ArrayList();
+    }
+
+    void Update()
+    {
+    	if (inside.Count > 0)
+    	{
+			// Get rune component
+        	var rune = transform.parent.GetComponent<Rune>();
+
+        	// Get direction vector of the rune
+        	Vector2 v = rune.GetVector();
+
+        	// Loop through all entities that have entered the effect area
+    		IEnumerator i = inside.GetEnumerator();
+    		while (i.MoveNext())
+    		{
+    			GameObject g = (GameObject)i.Current;
+
+    			// Calculate distance from entity to base
+        		float dist = Vector2.Distance(rune.GetPosition(), g.transform.position);
+        		// Scale force by that distance
+        		float thrust = force;
```

Comment:
It doesn't look there's any scaling going on here?

Reply:
Oh that comment is outdated, I was going to use the distance but then opted for just applying the same amount of force every frame. Will eventually use that code, but I just removed it for now. Only has line 35 now